packages/lending/src/protocols/radiant-v2/lending-protocol.test.ts

```diff
@@ -14,20 +15,13 @@ describe('Test Radiant V2 LendingProtocol', function () {
           chainId: 1,
           protocolId: 'radiant-v2',
           marketId: 'mainnet',
-          utilization: '0.97822902811306442548',
-          healthRate: '1.06588844353662664751',
-          netAPY: '-1.19664219493855548459',
-          totalSupplyUSD: '11342075.4944172804881341603106315',
-          totalBorrowUSD: '8649160.64068362879503470404861',
+          utilization: '0.88969606477533475173',
+          healthRate: '1.17098530871127146488',
+          totalSupplyUSD: '14802936.73275448119034789256139326',
+          totalBorrowUSD: '10237301.2493721107694920432157276',
           supplies: [
             {
-              token: {
-                chainId: 1,
-                address: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
-                decimals: 6,
-                symbol: 'USDT',
-                name: 'Tether USD',
-              },
+              token: mainnetTokens.USDT,
```

Comment:
Yeah. I think it's better for maintenance because

1. It shrink the size of `lending-protocol.test.ts` and make it easier to read
2. I understand it is easier to copy paste from the log at the beginning, but when we need to add a new token into this file, it would need to manually key-in the data to avoid clause issue.
3. The other benefit will be checking the info in the token mapping again in case of any typo.

Reply:
OK. c.c. @LuPoYi this change.